partner_risk_insurance_security/views/res_partner_view.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!--Add risk insurance tab in partner form. -->
+    <record model="ir.ui.view" id="view_partner_form">
+        <field name="name">res.partner.form.risk_insurance_security</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="priority" eval="50" />
```

Comment:
**issue**: The fields you're modifying do not exist in `base.view_partner_form`. I think you want to inherit from `partner_risk_insurance.view_partner_form`.

**suggestion** You can also drop the `priority` IMO, if you don't need it for anything in particular.

Reply:
Sorry, I realize it was ambiguous. By "drop" I meant "remove" the line where you specify the priority entirely.

Anyway, non-blocking.